frontend/src/app/choreography/choreography.component.html

```diff
@@ -6,6 +6,7 @@
     <!--    </mat-form-field>-->
     <app-frame-manager
       (activeChanged)="activeFrame = $event"
+      (subActiveChanged)="activeSubframe = $event"
```

Comment:
**suggestion:** call these `activeFrameChange` and `activeSubframeChange` so not in past tense (compare `click`) and mention the domain object name